vuepress/docs/next/docs/curate/bundle-details.md

```diff
@@ -94,22 +94,24 @@ The following is a list of specifications for the bundle descriptor and its comp
 ```
 
 ### Microservices Specifications
-|Name|Type|Required|Possible Values|Description|
+|Name|Type|Required|Possible Values [default]|Description|
 |:-|:-|:-|:-|:------------------------|
 |`name`|String|Yes||Microservice name|
 |`stack`|Enum|Yes|*spring-boot<br/>*node<br/>*custom|Microservice stack |
 |`dbms`|Enum|No|*none^  *embedded  *postgresql  *mysql  |DBMS required by the MS to provide services|
-|`ingressPath`|String|No||Custom ingress path|
 |`healthCheckPath`|String|No||Endpoint for a health check|
+|`healthCheckIngress`|Enum| Yes |*[canonical]  *custom| Use the canonical address or customize with `ingressPath`
```

Comment:
I thought this change was backed out for 7.3. @firegloves am I correct?

Reply:
Yes @nshaw @jyunmitch, we have reverted the change, so it is no longer implemented

Reply:
@firegloves thanks!

Reply:
@nshaw, I'll just drop this PR altogether. that will keep the ingressPath as it was.

Reply:
@jyunmitch The language around oracle and external dbs was still useful but it's a pretty small change in the end.